assets/app/components/siteContainer.js

```diff
@@ -27,7 +27,7 @@ class SiteContainer extends React.Component {
     if (currentSite) {
       siteActions.fetchSiteConfigsAndAssets(currentSite);
     } else {
-      replaceRouterHistory('/sites');
+      replaceHistory('/sites');
```

Comment:
good catch! some day we should get some test coverage on this component. it's never surprising to find bugs in code that's missing tests.